homeassistant/components/rachio/switch.py

```diff
@@ -391,6 +416,7 @@ def __init__(self, person, controller, data, current_schedule):
         self._duration = data[KEY_DURATION]
         self._schedule_enabled = data[KEY_ENABLED]
         self._summary = data[KEY_SUMMARY]
+        self.type = data.get(KEY_TYPE)
```

Comment:
```suggestion
        self.type = data.get(KEY_TYPE, SCHEDULE_TYPE_FIXED)
```

Reply:
Does this need to stay exposed since its being accessed from:?

```
async def async_setup_entry(hass, config_entry, async_add_entities):
    """Set up the Rachio switches."""	    """Set up the Rachio switches."""
    # Add all zones from all controllers as switches	    has_flex_sched = False
    entities = await hass.async_add_executor_job(_create_entities, hass, config_entry)	    entities = await hass.async_add_executor_job(_create_entities, hass, config_entry)
    for entity in entities:
        if isinstance(entity, RachioSchedule) and entity.type == SCHEDULE_TYPE_FLEX:
```

Reply:
Yes.